Makefile

```diff
@@ -1,6 +1,6 @@
 VERSION = 1.0.9
 
-FLAGS = -dtypes -g
+FLAGS = -dtypes -g -annot -bin-annot
```

Comment:
`-dtypes` is the same as `-annot`. Since `-annot` has been around for a while now, it's safe to get rid of `-dtypes`.